protocol/src/order-event.md

```diff
@@ -47,9 +47,9 @@ Events are [addressable events](https://github.com/nostr-protocol/nips/blob/mast
 ## Tags
 
 - `d` < Order ID >: A unique identifier for the order.
-- `k` < Order type >: `sell` or `buy`.
-- `f` < Currency >: The asset being traded, using the [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217) standard.
-- `s` < Status >: `pending`, `canceled`, `in-progress`, `success`.
+- `k` < Order type >: `sell` or `buy`. This specifies the type of transaction in terms of bitcoin. "sell" means selling bitcoin, while "buy" indicates buying bitcoin.
+- `f` < Currency >: The fiat asset being traded, using the [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217) standard.
+- `s` < Status >: `active`, `canceled`, `canceled-by-admin`, `settled-by-admin`, `completed-by-admin`, `dispute`, `expired`, `fiat-sent`, `settled-hold-invoice`, `pending`, `success`, `waiting-buyer-invoice`, `waiting-payment`, `cooperatively-canceled`
```

Comment:
Mostro currently publishes more status than it should but that will be modified. To increase user privacy and follow pin 69 the statuses of orders published in the 38383 events must be just `pending`, `canceled`, `in-progress`, `success`.
If other extra statuses are maintained later, it will be for internal management in the mostro database, not publicly.

Reply:
Thanks for the explanation @Catrya ! I've reverted the status change and kept only the -k tag clarification. Happy to close this PR if it's not needed.